packages/nimble-components/src/rich-text-viewer/tests/rich-text-viewer.spec.ts

```diff
@@ -10,4 +31,211 @@ describe('RichTextViewer', () => {
+
+            await connect();
+
+            expect(pageObject.getChildTags()).toEqual(['P', 'A']);
+            expect(pageObject.getLastChildTagContents()).toBe(
+                'https://nimble.ni.dev/'
+            );
+            expect(pageObject.getLastChildAttribute('href')).toBe(
+                'https://nimble.ni.dev/'
+            );
+        });
+
+        it('should convert numbered list with bold markdown string to "ol", "li" and "strong" HTML tags', async () => {
+            element.markdown = '1. **Numbered list in bold**';
+
+            await connect();
+
+            expect(pageObject.getChildTags()).toEqual([
+                'OL',
+                'LI',
+                'P',
+                'STRONG'
+            ]);
+            expect(pageObject.getLastChildTagContents()).toBe(
+                'Numbered list in bold'
+            );
+        });
+
+        it('should convert bulleted list with italics markdown string to "ul", "li" and "em" HTML tags', async () => {
+            element.markdown = '* *Bulleted list in italics*';
+
+            await connect();
+
+            expect(pageObject.getChildTags()).toEqual(['UL', 'LI', 'P', 'EM']);
+            expect(pageObject.getLastChildTagContents()).toBe(
+                'Bulleted list in italics'
+            );
+        });
+
+        it('should convert bulleted list with direct links markdown string to "ul", "li" and "a" HTML tags', async () => {
+            element.markdown = '* <https://nimble.ni.dev/>';
+
+            await connect();
+
+            expect(pageObject.getChildTags()).toEqual(['UL', 'LI', 'P', 'A']);
+            expect(pageObject.getLastChildTagContents()).toBe(
+                'https://nimble.ni.dev/'
+            );
+            expect(pageObject.getLastChildAttribute('href')).toBe(
+                'https://nimble.ni.dev/'
+            );
+        });
+
+        it('should convert direct links in bold markdown string to "strong" and "a" HTML tags', async () => {
+            element.markdown = '**<https://nimble.ni.dev/>**';
+
+            await connect();
+
+            expect(pageObject.getChildTags()).toEqual(['P', 'STRONG', 'A']);
+            expect(pageObject.getLastChildTagContents()).toBe(
+                'https://nimble.ni.dev/'
+            );
+            expect(pageObject.getLastChildAttribute('href')).toBe(
+                'https://nimble.ni.dev/'
+            );
+        });
+    });
+
+    describe('various not supported markdown string values render as expected', () => {
+        const notSupportedMarkdownStrings: { name: string }[] = [
+            { name: '> blockquote' },
+            { name: '`code`' },
+            { name: '```fence```' },
+            { name: '# Heading' },
+            { name: '[link](url)' },
+            { name: '[ref][link] [link]:url' },
+            { name: '![Text](Image)' },
+            { name: '&nbsp;' },
+            { name: '---' },
+            { name: '<div><p>text</p></div>' },
```

Comment:
Could you add a bit more coverage for other HTML syntax that people might try?
```suggestion
            { name: '<div><p>text</p></div>' },
            { name: '<b>not bold</b>' },
            { name: '<em>not italic</em>' },
```